2025-09-18 07:21:09.036 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration daikin_onecta which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2025-09-18 07:21:09.037 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2025-09-18 07:21:09.037 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration google_assistant_sdk_custom which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2025-09-18 07:21:12.619 ERROR (MainThread) [pydaikin.daikin_brp084] Error communicating with device: 404, message='Not Found', url='http://192.168.10.54/dsiot/multireq'
2025-09-18 07:21:13.992 WARNING (MainThread) [homeassistant.components.light] light.testdimmer (<class 'homeassistant.components.mqtt.light.schema_json.MqttLightJson'>) does not report a color mode, this will stop working in Home Assistant Core 2025.3, please create a bug report at https://github.com/home-assistant/core/issues?q=is%3Aopen+is%3Aissue+label%3A%22integration%3A+mqtt%22
2025-09-18 07:21:45.453 ERROR (MainThread) [homeassistant.components.automation.stue_klima_reguler_settpunkt_pa_daikin] Stue klima – reguler settpunkt på Daikin: Choose at step 4: choice 1: Error executing script. Invalid data for call_service at pos 2: value should be a string for dictionary value @ data['entity_id']
2025-09-18 07:21:45.453 ERROR (MainThread) [homeassistant.components.automation.stue_klima_reguler_settpunkt_pa_daikin] Stue klima – reguler settpunkt på Daikin: Error executing script. Invalid data for choose at pos 4: value should be a string for dictionary value @ data['entity_id']
2025-09-18 07:21:45.453 ERROR (MainThread) [homeassistant.components.automation.stue_klima_reguler_settpunkt_pa_daikin] Error while executing automation automation.stue_klima_reguler_settpunkt_pa_daikin: value should be a string for dictionary value @ data['entity_id']
2025-09-18 07:21:45.456 ERROR (MainThread) [homeassistant.components.automation.stue_klima_reguler_settpunkt_pa_daikin] Stue klima – reguler settpunkt på Daikin: Choose at step 4: choice 1: Error executing script. Invalid data for call_service at pos 2: value should be a string for dictionary value @ data['entity_id']
2025-09-18 07:21:45.456 ERROR (MainThread) [homeassistant.components.automation.stue_klima_reguler_settpunkt_pa_daikin] Stue klima – reguler settpunkt på Daikin: Error executing script. Invalid data for choose at pos 4: value should be a string for dictionary value @ data['entity_id']
2025-09-18 07:21:45.456 ERROR (MainThread) [homeassistant.components.automation.stue_klima_reguler_settpunkt_pa_daikin] Error while executing automation automation.stue_klima_reguler_settpunkt_pa_daikin: value should be a string for dictionary value @ data['entity_id']
2025-09-18 07:21:54.729 WARNING (MainThread) [homeassistant.components.mqtt.light.schema_json] Invalid color mode 'color_temp' received for entity light.kjokken_zb_lys
2025-09-18 07:21:58.729 ERROR (MainThread) [homeassistant.components.automation.stue_klima_reguler_settpunkt_pa_daikin] Stue klima – reguler settpunkt på Daikin: Choose at step 4: choice 1: Error executing script. Invalid data for call_service at pos 2: value should be a string for dictionary value @ data['entity_id']
2025-09-18 07:21:58.730 ERROR (MainThread) [homeassistant.components.automation.stue_klima_reguler_settpunkt_pa_daikin] Stue klima – reguler settpunkt på Daikin: Error executing script. Invalid data for choose at pos 4: value should be a string for dictionary value @ data['entity_id']
2025-09-18 07:21:58.730 ERROR (MainThread) [homeassistant.components.automation.stue_klima_reguler_settpunkt_pa_daikin] Error while executing automation automation.stue_klima_reguler_settpunkt_pa_daikin: value should be a string for dictionary value @ data['entity_id']
2025-09-18 07:22:00.390 ERROR (MainThread) [homeassistant.components.automation.stue_klima_reguler_settpunkt_pa_daikin] Stue klima – reguler settpunkt på Daikin: Choose at step 4: choice 1: Error executing script. Invalid data for call_service at pos 2: value should be a string for dictionary value @ data['entity_id']
2025-09-18 07:22:00.391 ERROR (MainThread) [homeassistant.components.automation.stue_klima_reguler_settpunkt_pa_daikin] Stue klima – reguler settpunkt på Daikin: Error executing script. Invalid data for choose at pos 4: value should be a string for dictionary value @ data['entity_id']
2025-09-18 07:22:00.392 ERROR (MainThread) [homeassistant.components.automation.stue_klima_reguler_settpunkt_pa_daikin] Error while executing automation automation.stue_klima_reguler_settpunkt_pa_daikin: value should be a string for dictionary value @ data['entity_id']
2025-09-18 07:22:29.263 ERROR (MainThread) [homeassistant.components.automation.stue_klima_reguler_settpunkt_pa_daikin] Stue klima – reguler settpunkt på Daikin: Choose at step 4: choice 1: Error executing script. Invalid data for call_service at pos 2: value should be a string for dictionary value @ data['entity_id']
2025-09-18 07:22:29.264 ERROR (MainThread) [homeassistant.components.automation.stue_klima_reguler_settpunkt_pa_daikin] Stue klima – reguler settpunkt på Daikin: Error executing script. Invalid data for choose at pos 4: value should be a string for dictionary value @ data['entity_id']
2025-09-18 07:22:29.264 ERROR (MainThread) [homeassistant.components.automation.stue_klima_reguler_settpunkt_pa_daikin] Error while executing automation automation.stue_klima_reguler_settpunkt_pa_daikin: value should be a string for dictionary value @ data['entity_id']
2025-09-18 07:22:29.950 ERROR (MainThread) [homeassistant.components.automation.stue_klima_reguler_settpunkt_pa_daikin] Stue klima – reguler settpunkt på Daikin: Choose at step 4: choice 1: Error executing script. Invalid data for call_service at pos 2: value should be a string for dictionary value @ data['entity_id']
2025-09-18 07:22:29.950 ERROR (MainThread) [homeassistant.components.automation.stue_klima_reguler_settpunkt_pa_daikin] Stue klima – reguler settpunkt på Daikin: Error executing script. Invalid data for choose at pos 4: value should be a string for dictionary value @ data['entity_id']
2025-09-18 07:22:29.951 ERROR (MainThread) [homeassistant.components.automation.stue_klima_reguler_settpunkt_pa_daikin] Error while executing automation automation.stue_klima_reguler_settpunkt_pa_daikin: value should be a string for dictionary value @ data['entity_id']
2025-09-18 07:22:29.956 ERROR (MainThread) [homeassistant.components.automation.stue_klima_reguler_settpunkt_pa_daikin] Stue klima – reguler settpunkt på Daikin: Choose at step 4: choice 1: Error executing script. Invalid data for call_service at pos 2: value should be a string for dictionary value @ data['entity_id']
2025-09-18 07:22:29.956 ERROR (MainThread) [homeassistant.components.automation.stue_klima_reguler_settpunkt_pa_daikin] Stue klima – reguler settpunkt på Daikin: Error executing script. Invalid data for choose at pos 4: value should be a string for dictionary value @ data['entity_id']
2025-09-18 07:22:29.957 ERROR (MainThread) [homeassistant.components.automation.stue_klima_reguler_settpunkt_pa_daikin] Error while executing automation automation.stue_klima_reguler_settpunkt_pa_daikin: value should be a string for dictionary value @ data['entity_id']
2025-09-18 07:22:42.821 ERROR (MainThread) [homeassistant.components.automation.stue_klima_reguler_settpunkt_pa_daikin] Stue klima – reguler settpunkt på Daikin: Choose at step 4: choice 1: Error executing script. Invalid data for call_service at pos 2: value should be a string for dictionary value @ data['entity_id']
2025-09-18 07:22:42.821 ERROR (MainThread) [homeassistant.components.automation.stue_klima_reguler_settpunkt_pa_daikin] Stue klima – reguler settpunkt på Daikin: Error executing script. Invalid data for choose at pos 4: value should be a string for dictionary value @ data['entity_id']
2025-09-18 07:22:42.823 ERROR (MainThread) [homeassistant.components.automation.stue_klima_reguler_settpunkt_pa_daikin] Error while executing automation automation.stue_klima_reguler_settpunkt_pa_daikin: value should be a string for dictionary value @ data['entity_id']
2025-09-18 07:23:14.365 ERROR (MainThread) [homeassistant.components.automation.stue_klima_reguler_settpunkt_pa_daikin] Stue klima – reguler settpunkt på Daikin: Choose at step 4: choice 1: Error executing script. Invalid data for call_service at pos 2: value should be a string for dictionary value @ data['entity_id']
2025-09-18 07:23:14.366 ERROR (MainThread) [homeassistant.components.automation.stue_klima_reguler_settpunkt_pa_daikin] Stue klima – reguler settpunkt på Daikin: Error executing script. Invalid data for choose at pos 4: value should be a string for dictionary value @ data['entity_id']
2025-09-18 07:23:14.369 ERROR (MainThread) [homeassistant.components.automation.stue_klima_reguler_settpunkt_pa_daikin] Error while executing automation automation.stue_klima_reguler_settpunkt_pa_daikin: value should be a string for dictionary value @ data['entity_id']
2025-09-18 07:25:02.210 ERROR (MainThread) [homeassistant.components.automation.stue_klima_reguler_settpunkt_pa_daikin] Stue klima – reguler settpunkt på Daikin: Choose at step 4: choice 1: Error executing script. Invalid data for call_service at pos 2: value should be a string for dictionary value @ data['entity_id']
2025-09-18 07:25:02.210 ERROR (MainThread) [homeassistant.components.automation.stue_klima_reguler_settpunkt_pa_daikin] Stue klima – reguler settpunkt på Daikin: Error executing script. Invalid data for choose at pos 4: value should be a string for dictionary value @ data['entity_id']
2025-09-18 07:25:02.213 ERROR (MainThread) [homeassistant.components.automation.stue_klima_reguler_settpunkt_pa_daikin] Error while executing automation automation.stue_klima_reguler_settpunkt_pa_daikin: value should be a string for dictionary value @ data['entity_id']
